cmake/mkvk.cmake

```diff
@@ -2,6 +2,16 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Code generation scripts that require a Vulkan SDK installation
+
```

Comment:
Do you think we should change the code generation scripts to use Vulkan_INCLUDE_DIRS instead of VULKAN_SDK? If would mean moving find_package(Vulkan) from vkloadtests.cmake and probably changes to the scripts.

Reply:
`mkvkformatfiles` still needs a `VULKAN_SDK` environment variable.

Reply:
Yes. Changes to the scripts will likely be necessary. mkvkformatfiles vulkan_core.h nothing else. I'm sure we could make it work from Vulkan_INCLUDE_DIRS. I will look at it later to confirm but IIRC you pass it the path of the file so it is the mkvk.cmake that is using $VULKAN_SDK to provide that path.

Reply:
It's OK to have more than one `find_package(Vulkan)` because the result is cached so later call don't search again.
Also `mkvkformatfiles` only needs `VULKAN_SDK` to find the include directory, so I changed it to use `Vulkan_INCLUDE_DIR` directly instead.
I think it's better to use the `Vulkan_INCLUDE_DIR` and `Vulkan_LIBRARY` (singular form) output variables from `find_package` because they won't be lists, so doing something like `cmake_path(REMOVE_FILENAME ${Vulkan_LIBRARY} OUTPUT VARIABLE Vulkan_LIBRARY_DIR)` is safer.
With these changes and those in `vkloadtests.cmake`, there are no more references to `VULKAN_SDK`, except for the default `MOLTEN_VK_SDK` value.